### ISSUE TITLE
Camera access for file drops

### ISSUE THREAD

**Issue by Noctudeit** [nextcloud/server#31234]
It would be great if someone could add a camera button to the file drop interface. 

As it stands, the person uploading files has two options; drag-and-drop and browse for files. Increasingly, people are responding to file requests from phones and other mobile devices which intentionally obscure their file structure from the end user making both of those options difficult and cumbersome to use. Additionally, document/image scanners are becoming less and less common which makes it harder for people to send scans via file drop. Both issues could be addressed with a camera button on the file drop web interface so the user can just snap a few pictures and have them automatically staged to be uploaded.

**Comment by acsfer** [comment]
Are you talking about mobile clients?

**Comment by Noctudeit** [comment]
Yes. I request sensitive documents from clients who don't have access to a scanner. I have walked some of them through the use of a mobile scanner app and then uploading the resulting files to NextCloud via a file drop link, but the process is tedious and outright impossible for some of my older clients. Many then result to just emailing the documents which raises data security concerns. 
I'm sure all of them could use file drop links if it could activate the device camera and automatically stage the images captured for upload.

**Comment by susnux** [comment]
For the webUI I think this does not make much sense. As the upload will usually (at least Android) ask you which app to use and offers you also the camera app.

So for the webUI it would only work if we use WebRTC to let the browser request an image, but this will mostly not work the way intended.

**For the apps this is already implemented, you can select "choose from camera" directly from within the app**
